src/main/java/com/facebook/openwifirrm/ucentral/gw/models/DeviceWithStatus.java

```diff
@@ -15,6 +15,7 @@
 public class DeviceWithStatus {
 	public String owner;
 	public String location;
+	/** Unreliable */
```

Comment:
??

Reply:
In an earlier review, you said not to use `device.venue` in `ConfigManager` and instead to use `deviceDataManager.getDeviceZone(device.serialNumber)`

What was the reason for that?

Reply:
We discussed. It was because we may not always be syncing with prov (it may not even be deployed)